app/setupPreloadAPIs.js

```diff
@@ -14,11 +14,79 @@ window.eval = global.eval = () =>
     throw new Error( 'Sorry, peruse does not support window.eval().' );
 };
 
+const nullifySafeAppAuthFuncs = ( app ) =>
+{
+    logger.silly( 'Nullifying props of returned safe app' );
+
+    // QUESTION: LoginForTest?
+    const safeAuthToNullify = ['genAuthUri', 'genShareMDataUri', 'genConnUri', 'openUri'];
```

Comment:
I don't think we need to nullify the functions which generate auth URI's, they are local and I don't see a problem with exposing them here if the app wants to perform that step individually. `openUri` needs to be wrapped rather than nullified as it should use the IPC channel instead of system_uri.